mount-helper/install/install.sh

```diff
@@ -138,7 +138,7 @@ _remove_apps() {
             fi
             apt-get purge -y --no-auto-remove "$app"
         elif is_linux LINUX_RED_HAT; then
-            if [[ $VERSION == 8.8 || $VERSION == 8.9 ]]; then
+            if [[ $VERSION == 8.8 || $VERSION == 8.9 || $VERSION == 8.10 ]]; then
```

Comment:
can we have a single method to check the version and use this method at other places so that we will not miss checking supported versions

Reply:
We are planning to support other versions  as well. When we get to it, I will create a. single method at make sure everything is in one place?